rules/validator.go

```diff
@@ -163,6 +155,21 @@ func (v *validator) validateFilters(f map[string]string) error {
 	return nil
 }
 
+func (v *validator) validatePolicies(ruleName string, policies []policy.Policy, types []metric.Type) error {
+	if len(policies) == 0 {
+		return fmt.Errorf("rule %s has no storage policies", ruleName)
```

Comment:
nit: Perhaps just say "has no policies"? A policy contains a storage policy and aggregation types, and users probably don't know what storage policies mean.